File: Python/Problem6.py
'''
Created on Aug 16, 2014

@author: Prathyush
'''

import math
from main import test

print "Difference between the sum of the squares of the first one hundred natural numbers and the square of the sum"
SUM1=0
SUM2=0
start=test.Time()

for i in xrange(1,100+1):
    SUM1+=i
    SUM2+=i**2
SUM1=SUM1**2




print SUM1-SUM2


end=test.Time()

print 'The test elapsed is: ' +str(end-start)+ 's'

File: Java/Problem6.java
public class Problem6 {

	public static void main(String v[]) {
		double start = System.nanoTime();
		int sum = 0, sum1 = 0;
		System.out
				.println("Difference between the sum of the squares of the first one hundred natural numbers and the square of the sum");
		for (int i = 1; i < 101; i++) {
			sum += i;
			sum1 += Math.pow(i, 2);

		}
		sum = (int) Math.pow(sum, 2);
		System.out.println(sum - sum1);
		double end = System.nanoTime();
		System.out.println("Time Elapsed is: " + String.valueOf((end - start))
				+ "ns");
	}
}
